### PR TITLE
README: moar :heart: moar readme :heart: and moar emoji

### PR DESCRIPTION
**Pull request by the-j0k3r**
moar :heart: the readme has a :heartbeat: 

@silverwind :medal_sports: :1st_place_medal:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,105 +26,106 @@
...
-* 🎨 Stylish for [Firefox](https://addons.mozilla.org/en-US/firefox/addon/2108/), [Chrome](https://chrome.google.com/extensions/detail/fjnbnpbmkenffdnngjfgmeleoegfcffe), [Opera](https://addons.opera.com/en/extensions/details/stylish/), [Safari](http://sobolev.us/stylish/) and [Firefox Mobile](https://addons.mozilla.org/en-US/firefox/addon/2108/).
-* 🎨 FreeStyler for [Chrome](https://chrome.google.com/webstore/detail/freestyler/hihigldmabkodfpehkgdemjklmaebmca).
+🎨 Stylus for [Firefox](https://addons.mozilla.org/en-US/firefox/addon/styl-us/), [Chrome](https://chrome.google.com/webstore/detail/stylus/clngdbkpkpeebahjckkjfobafhncgmne) and [Opera](https://addons.opera.com/en-gb/extensions/details/stylus/).
+🎨 Stylish for [Firefox](https://addons.mozilla.org/en-US/firefox/addon/2108/), [Chrome](https://chrome.google.com/extensions/detail/fjnbnpbmkenffdnngjfgmeleoegfcffe), [Opera](https://addons.opera.com/en/extensions/details/stylish/), [Safari](http://sobolev.us/stylish/) and [Firefox Mobile](https://addons.mozilla.org/en-US/firefox/addon/2108/).
+🎨 FreeStyler for [Chrome](https://chrome.google.com/webstore/detail/freestyler/hihigldmabkodfpehkgdemjklmaebmca).
```
Comment (Mottie) [inline README.md]: This breaks the list... click on the rich diff icon to see.



I actually like the bullets in front, but if you want to remove them, you'll probably need to add a `<br>` to the end of each entry.

File: README.md [hunk 2]
```diff
@@ -26,105 +26,106 @@
...
+:eight_spoked_asterisk: `grunt themes`: Parse all theme files and combine them into minified styles in the `themes` directory.
+:eight_spoked_asterisk: `grunt clean`: Reformat `github-dark.css` to conform the style guide.
+:eight_spoked_asterisk: `grunt lint`: Run Stylelint on `github-dark.css`.
+:eight_spoked_asterisk: `grunt authors`: Regenerate the `AUTHORS` file based on git history.
+:eight_spoked_asterisk: `grunt imagemin`: Minify all images present in the `image` directory.
+:eight_spoked_asterisk: `grunt generate`: Regenerate auto-generated CSS rules based on GitHub's stylesheet. This should be ran regularly.
+:eight_spoked_asterisk: `grunt patch`: Increment the version on the patch version number. Also updates headers and creates a commit.
+:eight_spoked_asterisk: `grunt minor`: Increment the version on the minor version number. Also updates headers and creates a commit.
+:eight_spoked_asterisk: `grunt major`: Increment the version on the major version number. Also updates headers and creates a commit.
+:eight_spoked_asterisk: `grunt update`: Update and install dependencies.
```
Comment (Mottie) [inline README.md]: Breaks the list and looks fugly.

Comment (Mottie) [reply README.md]: Maybe a 🙈 would be more appropriate? LOL

Comment (the-j0k3r) [reply README.md]: :+1:

File: README.md [hunk 3]
```diff
@@ -26,105 +26,106 @@
...
+| Pastel on Dark             | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Solarized Dark             | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Terminal                   | :heavy_check_mark: |                    |                    |
+| Tomorrow Night             | :heavy_check_mark: |                    | :heavy_check_mark: |
+| Tomorrow Night Blue        | :heavy_check_mark: |                    | :heavy_check_mark: |
+| Tomorrow Night Bright      | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Tomorrow Night Eighties    | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Twilight (**default**)     | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Vibrant Ink                | :heavy_check_mark: | :heavy_check_mark: |                    |
+
```
[the remark this answers is not in the document]
Comment (the-j0k3r) [reply README.md]: have some :cake:

File: README.md (hunk 1 above)
Comment (the-j0k3r) [reply README.md]: added breaks :) lists fixed, the editor config was stripping my markdown breaks, fixed that also.

File: README.md (hunk 2 above)
Comment (the-j0k3r) [reply README.md]: :monkey: it was

File: README.md [hunk 4]
```diff
@@ -26,105 +26,106 @@
...
-* 🎨 Stylish for [Firefox](https://addons.mozilla.org/en-US/firefox/addon/2108/), [Chrome](https://chrome.google.com/extensions/detail/fjnbnpbmkenffdnngjfgmeleoegfcffe), [Opera](https://addons.opera.com/en/extensions/details/stylish/), [Safari](http://sobolev.us/stylish/) and [Firefox Mobile](https://addons.mozilla.org/en-US/firefox/addon/2108/).
-* 🎨 FreeStyler for [Chrome](https://chrome.google.com/webstore/detail/freestyler/hihigldmabkodfpehkgdemjklmaebmca).
+🎨 Stylus for [Firefox](https://addons.mozilla.org/en-US/firefox/addon/styl-us/), [Chrome](https://chrome.google.com/webstore/detail/stylus/clngdbkpkpeebahjckkjfobafhncgmne) and [Opera](https://addons.opera.com/en-gb/extensions/details/stylus/).  
+🎨 Stylish for [Firefox](https://addons.mozilla.org/en-US/firefox/addon/2108/), [Chrome](https://chrome.google.com/extensions/detail/fjnbnpbmkenffdnngjfgmeleoegfcffe), [Opera](https://addons.opera.com/en/extensions/details/stylish/), [Safari](http://sobolev.us/stylish/) and [Firefox Mobile](https://addons.mozilla.org/en-US/firefox/addon/2108/).  
+🎨 FreeStyler for [Chrome](https://chrome.google.com/webstore/detail/freestyler/hihigldmabkodfpehkgdemjklmaebmca).
```
Comment (Mottie) [inline README.md]: *cringe* I usually end up switching between two different text editors. One obeys the `.editorconfig` and the other doesn't... in either case, I'm always having issues with the editors removing trailing spaces. Would you please change the trailing space to a `<br>`?

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Mottie) [reply README.md]: The 🍰is a lie!

File: README.md [hunk 5]
```diff
@@ -26,105 +26,106 @@
...
-| Tomorrow Night Bright      |   ◻️    |     🔹      |     🔸    |
-| Tomorrow Night Eighties    |   ◻️    |     🔹      |     🔸    |
-| Twilight (**default**)     |   ◻️    |     🔹      |     🔸    |
-| Vibrant Ink                |   ◻️    |     🔹      |          |
-
-* Support for [Codemirror](https://codemirror.net/demo/theme.html) and [Jupyter notebook syntax highlighting](https://github.com/sujitpal/statlearning-notebooks/blob/master/src/chapter2.ipynb) themes as listed above.
-* Please provide a pull request if you have or want to create a missing theme.
+| Theme                      |       GitHub       |     CodeMirror     |      Jupyter       |
+|----------------------------| :----------------: | :----------------: | :----------------: |
+| Ambiance                   | :heavy_check_mark: | :heavy_check_mark: |                    |
```
Comment (silverwind) [inline README.md]: Please use unicode characters so it looks good outside GitHub too

File: README.md (hunk 4 above)
Comment (the-j0k3r) [reply README.md]: :cry:

File: README.md (hunk 5 above)
Comment (the-j0k3r) [reply README.md]: :sob:

File: README.md [hunk 6]
```diff
@@ -26,105 +26,106 @@
...
+| Tomorrow Night             | :heavy_check_mark: |                    | :heavy_check_mark: |
+| Tomorrow Night Blue        | :heavy_check_mark: |                    | :heavy_check_mark: |
+| Tomorrow Night Bright      | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Tomorrow Night Eighties    | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Twilight (**default**)     | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Vibrant Ink                | :heavy_check_mark: | :heavy_check_mark: |                    |
+
+> Support for [Codemirror](https://codemirror.net/demo/theme.html) and [Jupyter notebook](https://github.com/sujitpal/statlearning-notebooks/blob/master/src/chapter2.ipynb) syntax highlighting themes as listed above.
+
+:monkey_face: Please provide a pull request if you have or want to create a missing theme.
```
Comment (silverwind) [inline README.md]: This one might be a bit excessive. Let's remove it

File: README.md [hunk 7]
```diff
@@ -26,105 +26,106 @@
...
+:monkey_face: `grunt themes`: Parse all theme files and combine them into minified styles in the `themes` directory.  
+:monkey_face: `grunt clean`: Reformat `github-dark.css` to conform the style guide.  
+:monkey_face: `grunt lint`: Run Stylelint on `github-dark.css`.  
+:monkey_face: `grunt authors`: Regenerate the `AUTHORS` file based on git history.  
+:monkey_face: `grunt imagemin`: Minify all images present in the `image` directory.  
+:monkey_face: `grunt generate`: Regenerate auto-generated CSS rules based on GitHub's stylesheet. This should be ran regularly.  
+:monkey_face: `grunt patch`: Increment the version on the patch version number. Also updates headers and creates a commit.  
+:monkey_face: `grunt minor`: Increment the version on the minor version number. Also updates headers and creates a commit.  
+:monkey_face: `grunt major`: Increment the version on the major version number. Also updates headers and creates a commit.  
+:monkey_face: `grunt update`: Update and install dependencies.
```
Comment (silverwind) [inline README.md]: Also excessive here,  I think the old list ist fine in this section

File: README.md (hunk 4 above)
Comment (silverwind) [reply README.md]: Agree, trailing whitespace is weird. Overall, I'm not sure I see a benefit of removing the lists. I kind of like the indentation it adds and the list marks aren't really an issue. I'd revert this type of change.

File: README.md (hunk 7 above)
Comment (the-j0k3r) [reply README.md]: mmm blame the monkey...

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (the-j0k3r) [reply README.md]: I thought cake made everyone feel better. :man_shrugging:

File: README.md (hunk 4 above)
Comment (the-j0k3r) [reply README.md]: I agree that using trailing spaces for this is weird but not for me to reason why the GH powers that be decide on some things and not others :man_shrugging: Must be Monday morning pre coffee decisions.

**Comment by the-j0k3r** [comment]
:see_no_evil: shameless, anything to do here? @Mottie

**Comment by Mottie** [comment]
Replace the two `>` that were added. I think it looks better as a list.

**Comment by the-j0k3r** [comment]
@Mottie done

File: README.md [hunk 8]
```diff
@@ -104,7 +104,7 @@ Then install using one of these methods:
 If you would like to contribute to this repository, please...
 
 1. 👓 Read the [contribution guidelines](./.github/CONTRIBUTING.md).
-1. ⛕ [Fork](https://github.com/StylishThemes/GitHub-Dark/fork) or ⬇️  [download](https://github.com/StylishThemes/GitHub-Dark/archive/master.zip) this repository.
+1. ![repo-forked](https://user-images.githubusercontent.com/136959/42383736-c4cb0db8-80fd-11e8-91ca-12bae108bccc.png) [Fork](https://github.com/StylishThemes/GitHub-Dark/fork) or ![cloud-download](https://user-images.githubusercontent.com/136959/42401932-9ee9cae0-813d-11e8-8691-16e29a85d3b9.png) [download](https://github.com/StylishThemes/GitHub-Dark/archive/master.zip) this repository.
 1. 👌 Create a pull request!
```
Comment (the-j0k3r) [inline README.md]: very clever, ill add this to notes

**Pull request merged**